charts/aws-efs-csi-driver/values.yaml

```diff
@@ -11,7 +11,7 @@ useFIPS: false
 
 image:
   repository: amazon/aws-efs-csi-driver
```

Comment:
**issue (blocking):** this needs to be changed to `public.ecr.aws/efs-csi-driver/amazon/aws-efs-csi-driver` as mentioned in [this issue](https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/1330)